liquidity/liquidity.go

```diff
@@ -274,10 +413,10 @@ func makeLoopOutRequest(suggestion *LoopOutRecommendation) loop.OutRequest {
 		},
 		MaxPrepayRoutingFee: prepayMaxFee,
 		MaxSwapRoutingFee:   routeMaxFee,
-		MaxMinerFee:         defaultMaximumMinerFee,
-		MaxSwapFee:          maxSwapFee,
-		MaxPrepayAmount:     defaultMaximumPrepay,
-		SweepConfTarget:     loop.DefaultSweepConfTarget,
+		MaxMinerFee:         quote.MinerFee,
+		MaxSwapFee:          quote.SwapFee,
+		MaxPrepayAmount:     quote.PrepayAmount,
```

Comment:
Or should just our max values be used here (not the quote)? Especially with the `MinerFee` using the quote takes away all the leeway that we had with the max.

Reply:
Makes sense for `MinerFee`, but I think using the quote value for swap fee/ prepay amount is ok? Values are unlikely to change

Reply:
Yes, those are ok, because the swap is only aborted if it can't happen for that price. The max miner fee is a security thing. We may not be able to get the htlc swept in time if chosen too low.

Is that factor 100x still somewhere for autoloop now?

Reply:
Split this out into sat/byte for our initial decision whether we want to even bother with suggestions and the max miner fee as discussed